README.md

```diff
@@ -176,8 +176,9 @@ The options are:
 * `-a sld` : slide reduction.
 * `-a cvp` : prints the vector in the lattice closest to the input vector.
 * `-v` : verbose mode. 
-* `-nolll` : does not apply to LLL-reduction. In the case of bkz, hkz and svp, by default, the input basis is LLL-reduced before anything else. This option allows to remove that initial LLL-reduction (note that other calls to LLL-reduction may occur during the execution).
+* `-nolll` : does not apply to LLL-reduction. In the case of bkz, hkz and svp, by default, the input basis is LLL-reduced before anything else. This option allows to remove that initial LLL-reduction (note that other calls to LLL-reduction may occur during the execution). In the cas of hlll, verify if the input basis is HLLL-reduced.
```

Comment:
I don't understand what this means.

Reply:
I will take an example: calling `fplll -a hlll -nolll [somebasis]` will not HLLL-reduce the basis, but just check if [somebasis] is HLLL-reduced or not. It is a way to only call `is_hlll_reduced` from the binary. I can rewrite this part to be more precise.

Reply:
Does it have the same behaviour when we do `-a lll -nolll`, i.e. is that consistent across the two LLLs?